vite.config.js

```diff
@@ -41,6 +41,9 @@ export default defineConfig({
     },
     server: {
         port: 5173,
+        watch: {
+            usePolling: true,
```

Comment:
```suggestion
            usePolling: process.platform === 'win32',
```

Reply:
can you please retest with the following change? 

we'd only want to introduce polling for windows users, all other platforms should still behave the same.

Reply:
Hi Aaron, 

I tried using the condition ( process.platform === 'win32' ) doesn't work (doesn't detect change in the src/ subfolder),  since inside my WSL2 is ubuntu, the process.platform return 'linux', but I changed to ( process.platform === 'linux' ) doesn't work either.

Reply:
Maybe, I will move to linux filesystem, since usePolling: true use higher CPU utilization

https://vitejs.dev/config/server-options.html

![image](https://github.com/okio-ai/nendo-web/assets/123477781/a1c94602-a47e-4c2f-bcc0-3c9c742a468e)

Reply:
Ok then that doesn't work. 
Then I can't merge that, because we don't want to introduce polling for all unix users of the platform. I will close the PR now, thanks for your contribution nonetheless. 

> Maybe, I will move to linux filesystem, since usePolling: true use higher CPU utilization
VSCode works great with WSL, I'm using pycharm with WSL which also works nicely, can recommend the switch!